packages/pages/src/common/src/template/internal/types.ts

```diff
@@ -71,6 +72,14 @@ export interface TemplateConfigInternal {
   slugField?: string;
 }
 
+/**
+ * The exported `config` function's definition.
+ */
+export interface WidgetConfigInternal {
```

Comment:
Can you make a src/widgets/internal/types.ts file instead of putting it in the template one?